pubspec.yaml

```diff
@@ -9,43 +9,47 @@ environment:
   sdk: ">=2.7.0 <3.0.0"
 
 dependencies:
-  connectivity_plus: ^1.0.2
-  cupertino_icons: ^1.0.2
-  date_time_picker: ^2.0.0
-  duration_picker: ^1.1.0+1
   flutter:
     sdk: flutter
```

Comment:
Here the sorted dependencies have been randomized again. I think keeping them sorted would be better as present in the codebase.

Reply:
I have Sorted dependencies in the latest commit.